charts/member-agent/templates/deployment.yaml

```diff
@@ -63,15 +75,16 @@ spec:
           volumeMounts:
           - name: provider-token 
             mountPath: /config
```

Comment:
do we need this volumn when CAAuth is enabled?

Reply:
Thanks for catching this up.

- changed the scope of the conditional bracket to ignore `provider-token` mounts and tested on local